draft-ietf-quic-transport.md

```diff
@@ -290,9 +290,38 @@ across changes in the client's and the server's network addresses.
 # Packet Types and Formats
 
 We first describe QUIC's packet types and their formats, since some are
-referenced in subsequent mechanisms.  Note that unless otherwise noted, all
-values specified in this document are in little-endian format and all field
-sizes are in bits.
+referenced in subsequent mechanisms.
+
+All values specified in this document are in big-endian format and all field
+sizes are in bits.  When discussing individual bits of fields, the least
+significant bit is referred to as bit 0.  Hexadecimal notation is used for
+describing the value of fields.
+
+
+## Versions
+
+QUIC versions are identified using a 32-bit value.
+
+The version 0x00000000 is reserved to represent an invalid version.  This
+version of the specification is identified by the number 0x00000001.
```

Comment:
I note, not as an objection, that I'll need to substantially change my Alt-Svc v= encoding to accommodate this format.  Thanks.  ;-)